generator/templates/drivers/inc/stm32f2xx.hpp.j2

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "static.hpp"
+
+#include "stm32f2xx_hal_can.h"
+#include <stdint.h>
+
+namespace CANlib {
+
+    typedef HAL_StatusTypeDef CANlib_Transmit_Error_T;
+
+    HAL_StatusTypeDef Init(uint32_t baudrate);
+    HAL_StatusTypeDef TransmitFrame(Frame& frame);
+}
```

Comment:
Perhaps we should remove this, as it's not going to work (it's missing some of the functions)